src/community/jwt-headers/jwt-headers-util/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ (c) 2024 Open Source Geospatial Foundation - all rights reserved
+  ~ This code is licensed under the GPL 2.0 license, available at the root
+  ~ application directory.
+  ~
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.geoserver.community</groupId>
+    <artifactId>jwt-headers</artifactId>
+    <version>2.26-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.geoserver.community.jwt-headers</groupId>
+  <artifactId>jwt-headers-util</artifactId>
+  <packaging>jar</packaging>
+  <name>JWT Headers Shared Library</name>
+
+  <dependencies>
+    <!--    <dependency>-->
```

Comment:
Please avoid commented-out code/configuration sections. Instead, have a comment here that no GeoServer dependency should be added (yes it's also in the README, I would not assume others to read it though, as there is no common practice to use READMEs to start with, in the GeoServer modules). It's a case where repeating oneself does not hurt.

Reply:
removed comments + added comment.